internal/constants.go

```diff
@@ -0,0 +1,52 @@
+package internal
+
+import (
+	"fmt"
+	submariner "github.com/submariner-io/submariner-operator/api/submariner/v1alpha1"
+	"github.com/submariner-io/submariner-operator/pkg/images"
+	"github.com/submariner-io/submariner-operator/pkg/names"
+	"github.com/submariner-io/submariner-operator/pkg/subctl/cmd/utils"
+	"strings"
+)
+
+const (
+	OperatorNamespace = "submariner-operator"
+)
+
+func OperatorImage(imageVersion string, repo string, imageOverrideArr []string) string {
```

Comment:
It seems awkward to have this directly under `internal` as users would then reference things as `internal.OperatorImage`, which doesn't  provide additional semantics as, ideally, the package name should. Also this file is named _constants_  but it mainly provides functions. Since the functions are related to images, I suggest naming it _image.go_ and putting it in an _internal/image_ package. This function can then be named `Operator` and referenced as `image.Operator`.

Reply:
Everything under `internal` will be private to `submariner-operator` and won't be available to users.

Reply:
There's still users, it doesn't matter if they're "private" to the project or not. It's about proper structure and readability.